.changeset/bright-hornets-enjoy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-frontend/application-config': minor
```

Comment:
I wouldn't consider this a feature but rather a fix.

```suggestion
'@commercetools-frontend/application-config': patch
```

Nit: also adjust the PR title please, thanks